.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,45 @@
-
-
 <!---
-If you haven't done so already, please read the CONTRIBUTING.md file to learn how we work and what we expect from all contributors.
+If you haven't done so already, please read the CONTRIBUTING.md file to learn
```

Comment:
I don't really like that this template is now huge. It's noise for any contributor which already contributed.

Reply:
> I don't really like that this template is now huge. It's noise for any contributor which already contributed.

On the other hand stuffing what we want into `CONTRIBUTING.md` has a proven track record of not to work...